src/header/Header.tsx

```diff
@@ -32,16 +33,19 @@ import './Header.scss';
 import { isResourceEditedState } from 'state/resourceEditedAtom';
 import { isFormOpenState } from 'state/formOpenAtom';
 import { handleActionIfFormOpen } from 'shared/components/UnsavedMessageBox/helpers';
+import { configFeaturesNames } from 'state/types';
 
 export function Header() {
   useAvailableNamespaces();
   const [isMenuOpen, setIsMenuOpen] = useState(false);
+  const [isSnowOpen, setIsSnowOpen] = useState(false);
```

Comment:
this should be stored in localStorage, because if we have a user who doesn't like this feature, and will click on top to disable snow, then after refresh, he needs to click the snow icon one more time if he doesn't want to see this, we need to have this stored somewhere in local memory.